porch/Makefile

```diff
@@ -159,7 +159,8 @@ run-local: porch
 	--standalone-debug-mode \
 	--kubeconfig="$(KUBECONFIG)" \
 	--cache-directory="$(CACHEDIR)" \
-	--function-runner 192.168.8.202:9445
+	--function-runner 192.168.8.202:9445 \
+	--repo-sync-frequency=60s
```

Comment:
Did we consider making this a property of the Repository object so it could be set to specific values for different repositories rather than a single value that must be specified at startup?

Reply:
Yes. The current change sets the default value while the property of repo can tweak individual repo sync. So, these two changes need not to be mutually exclusive. I did this change because it satisfies our use case with minimal changes.